next.config.js

```diff
@@ -5,6 +5,7 @@ const initExport = {
     graphqlUrl: process.env.INTERNAL_GRAPHQL_URL
   },
   publicRuntimeConfig: { // Will be available on both server and client
+    externalAssetsUrl: process.env.EXTERNAL_ASSETS_URL,
```

Comment:
Should add `EXTERNAL_ASSETS_URL=http://localhost:3000` to `.env.example`

Reply:
added